controller/adoptopenebs/jiva.go

```diff
@@ -0,0 +1,20 @@
+package adoptopenebs
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+// formJivaConfig forms the desired OpenEBS CR config for Jiva.
+func (p *Planner) formJivaConfig() error {
+	jivaConfig := &unstructured.Unstructured{
```

Comment:
can we refactor this to avoid jivaConfig local variable?
Same comments applies to other functions as well